gsad/src/gsad_http.c

```diff
@@ -465,38 +465,78 @@ char *
 gsad_message_new (const char *title, const char *msg,
                   cmd_response_data_t *response_data)
 {
-  gchar *xml, *resp;
-
-  xml = g_strdup_printf ("<gsad_response>"
-                         "<title>%s (GSA %s)</title>"
-                         "<message>%s</message>"
-                         "<token>%s</token>"
-                         "</gsad_response>",
-                         title,
-                         GSAD_VERSION,
-                         msg,
-                         "");
+  gchar *html;
+
+  html = g_strdup_printf (
+    "<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Transitional//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd\">"
```

Comment:
Since we don't use xslt anymore we can use html5 here. Just use <!DOCTYPE html> and remove the xmlns from the <html> tag.